src/utilities/logger.ts

```diff
@@ -20,8 +21,12 @@ export class Logger {
 		this.namespace = `vip-rtc${ localNamespace ? `:${ localNamespace }` : '' }`;
 	}
 
+	public setEnabled( enabled: boolean ): void {
+		this.enabled = enabled;
+	}
```

Comment:
How would this be used and why? Can distracted developers just filter their console?

It's not called and can't be called from the browser console. Can we remove this until we have a defined use case?